tasks/testwasher.py

```diff
@@ -0,0 +1,88 @@
+from collections import defaultdict
+import json
+import yaml
+from typing import List
+from tasks.test_core import ModuleTestResult
+
+
+class TestWasher:
+    test_output_json_file = "module_test_output.json"
+    flaky_test_indicator = "flakytest: this is a known flaky test"
+    flakes_file_path = "flakes.yaml"
+
+    known_flaky_tests = defaultdict(set)
+
+    def __init__(self, test_output_json_file=None, flaky_test_indicator=None, flakes_file_path=None):
+        self.test_output_json_file = test_output_json_file if test_output_json_file else "module_test_output.json"
```

Comment:
You could set `"module_test_output.json"` as the default value for the parameter directly